src/fluree/db/dbfunctions/core.cljc

```diff
@@ -30,6 +31,11 @@
   #?(:clj  (reset! db-fn-cache (cache/fifo-cache-factory {} :threshold 500))
      :cljs (reset! db-fn-cache (cache/lru-cache-factory {} :threshold 500))))
 
+(defn- read-str
+  "Like read-string but abstracts CLJ / CLJS differences"
+  [s]
+  #?(:clj  (edn/read-string s)
```

Comment:
This is the reader conditional you can eliminate if you `:refer [read-string]` on both platforms.

Reply:
excluded clojure.core read-string & consolidated read-string logic.